docs/app/api/og.png/route.tsx

```diff
@@ -16,8 +15,8 @@ const Container = ({ children }: { children: ReactNode }) => (
       position: 'relative',
       width: '100%',
       height: '100%',
-      backgroundColor: colors.bg.body,
-      color: colors.text.primary.DEFAULT,
+      backgroundColor: 'var(--color-bg-body)',
```

Comment:
does this work? :D Not sure if the CSS File is available here

Reply:
https://marigold-docs-git-dst-648upgrade-tailwindcss-marigold.vercel.app/api/og.png

(I think we need to hard code these here :-/)